### ISSUE TITLE
Error on :TSRename

### ISSUE THREAD

**Issue by paldepind** [mhartington/nvim-typescript#124]
I sometimes get this error when trying to rename something with `TSRename`.

![image](https://user-images.githubusercontent.com/521604/37608084-de4adc0a-2b99-11e8-990e-acfaa54e6f6b.png)

Unfortunately, I cannot find a way to reproduce it. When restarting Neovim the issue disappears.

**Comment by mhartington** [comment]
Yeah,  I've seen this before, but have been too busy to squash. Will look into it!

**Comment by paldepind** [comment]
Nice! Thank you 😄
